lib/main.coffee

```diff
@@ -22,6 +22,10 @@ module.exports =
       type: 'boolean'
       default: true
       description: 'When listing directory items, list subdirectories before listing files.'
+    alphanumericSort:
+      type: 'boolean'
+      default: true
+      description: 'By default files/folders are sorted alphanumerically, ticking this option will sort them based on ascii values, using String.localeCompare.'
```

Comment:
Again, is there a way to make this description a bit clearer? To me alphanumeric and ASCII sorting are pretty much the same thing. I think it warrants emphasis of the "natural" nature somehow.

```
Sort numbers and versions in natural rather than strict lexical order. For example, `1` would sort before `10` even though it's a shorter string.
```

Feel free to adjust my wording here to cover what natural-sort lite actually does.